.circleci/config.yml

```diff
@@ -200,6 +200,23 @@ jobs:
           paths:
             - "**/*"
 
+      # Since there is no UMD bundle that includes everything from the CDK, we need to move
+      # all bundles into a directory. This allows us to store all CDK UMD bundles as job
+      # artifacts that can be picked up by the Angular Github bot.
+      - run:
+          name: Prepare CDK artifacts for publish.
+          command: |
+            mkdir -p  /tmp/cdk-umd-artifacts
+            cp dist/releases/cdk/bundles/**/*.umd.js /tmp/cdk-umd-artifacts
```

Comment:
Looks like `dist/releases/cdk/bundles/**/*.umd.js` doesn't exist during the circle run.

Reply:
Yeah, I already force pushed. Thanks